content/sensu-enterprise/3.1/installation/overview.md

```diff
@@ -0,0 +1,204 @@
+instructions linked below, you'll need to obtain your Sensu Enterprise
+repository credentials via the [Sensu Account Manager
+portal][1]. These credentials are comprised of
+a username and password which are required to access Sensu Enterprise
+software packages.
+
+_**NOTE: manual installation is recommended for pre-production environments
+only.** Please note that this guide is not intended to provide instructions for
+deploying Sensu Enterprise into "production" environments. Production
+deployment strategies &ndash; including using automation tools like
+Chef, Puppet, or Ansible to install and configure Sensu &ndash; are
+be discussed [at the conclusion of the Sensu Core installation guide][2]._
+
+{{< platformBlock "RHEL/CentOS" >}}
+### Installing Sensu Enterprise and prerequisites on RHEL/Centos
+
+Sensu Enterprise builds on the same [architecture][arch] as Sensu Core.
+
+If you're a new Sensu user getting started with Sensu Enterprise,
+please complete the following selected sections of the [Sensu installation guide][3]:
+
+1. [Install and Configure Redis](/sensu-core/latest/installation/install-redis-on-rhel-centos)
+2. [Install and Configure RabbitMQ](/sensu-core/latest/installation/install-rabbitmq-on-rhel-centos)
+3. [Install Sensu Enterprise](/sensu-core/latest/platforms/sensu-on-rhel-centos/#sensu-enterprise)
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Ubuntu/Debian" >}}
+### Installing Sensu Enterprise and prerequisites on Ubuntu/Debian
+
+Sensu Enterprise builds on the same [architecture][arch] as Sensu Core.
+
+If you're a new Sensu user getting started with Sensu Enterprise,
+please complete the following selected sections of the [Sensu installation guide][3]:
+
+1. [Install and Configure Redis](/sensu-core/latest/installation/install-redis-on-ubuntu-debian)
+2. [Install and Configure RabbitMQ](/sensu-core/latest/installation/install-rabbitmq-on-ubuntu-debian)
+3. [Install Sensu Enterprise](/sensu-core/latest/platforms/sensu-on-ubuntu-debian/#sensu-enterprise)
+
+{{< platformBlockClose >}}
+
+## Upgrading from Sensu Core to Sensu Enterprise
+
+For those already running Sensu Core, Sensu Enterprise is designed to be a
+drop-in replacement for the Sensu Core server and API. Once installed,
+no configuration changes are required – simply terminate the
+`sensu-server` and `sensu-api` processes, and start the `sensu-enterprise`
+process to resume operation of your Sensu instance. Some configuration
+changes may be required to take advantage of [built-in integrations][4]
+or added-value features like [contact routing][5].
+
+{{< platformBlock "RHEL/CentOS" >}}
+### Upgrading from Sensu Core to Sensu Enterprise on RHEL/Centos
+
+If you're already a Sensu Core user, the following steps will guide
+you through a manual upgrade to Sensu Enterprise:
+
+1. Install the `sensu-enterprise` package as described in [Install Sensu Enterprise](/sensu-core/latest/platforms/sensu-on-rhel-centos/#sensu-enterprise).
```

Comment:
I got the following error when testing this (although this error is referring to the dashboard package)

```
Transaction check error:
  file /etc/sensu/dashboard.d/.keep from install of sensu-enterprise-dashboard-1:2.12.0-1.x86_64 conflicts with file from package uchiwa-1:1.3.1-1.x86_64
```

Then when I try `sudo service sensu-enterprise start`, I get `Failed to start sensu-enterprise.service: Unit not found.`

Reply:
Resolved this by reordering install steps both in this document and under `platforms` section.